src/dali/plugin/input/rest/kraken.py

```diff
@@ -226,174 +196,150 @@ def _compute_tx_set(self, trade_history, ledger, processed_transactions) -> List
+                    fiat_in_with_fee: str = trade_history[record[_REFID]][_COST]
                     result.append(
                         InTransaction(
-                            plugin=kwargs[Keyword.PLUGIN.value],
-                            unique_id=kwargs[Keyword.UNIQUE_ID.value],
-                            raw_data=kwargs[Keyword.RAW_DATA.value],
-                            timestamp=kwargs[Keyword.TIMESTAMP.value],
-                            asset=kwargs[Keyword.ASSET.value],
-                            exchange=kwargs[Keyword.EXCHANGE.value],
-                            holder=kwargs[Keyword.HOLDER.value],
-                            transaction_type=kwargs[Keyword.TRANSACTION_TYPE.value],
-                            spot_price=kwargs[Keyword.SPOT_PRICE.value],
-                            crypto_in=kwargs[Keyword.CRYPTO_IN.value],
-                            crypto_fee=kwargs[Keyword.CRYPTO_FEE.value],
-                            fiat_in_no_fee=kwargs[Keyword.FIAT_IN_NO_FEE.value],
-                            fiat_in_with_fee=kwargs[Keyword.FIAT_IN_WITH_FEE.value],
-                            fiat_fee=kwargs[Keyword.FIAT_FEE.value],
-                            notes=kwargs[Keyword.NOTES.value],
+                            plugin=self.__PLUGIN_NAME,
+                            unique_id=Keyword.UNKNOWN.value,
+                            raw_data=raw_data,
+                            timestamp=timestamp_value,
+                            asset=asset,
+                            exchange=self.__EXCHANGE_NAME,
+                            holder=self.account_holder,
+                            transaction_type=transaction_type,
+                            spot_price=spot_price,
+                            crypto_in=crypto_in,
+                            crypto_fee=crypto_fee,
+                            fiat_in_no_fee=fiat_in_no_fee,
+                            fiat_in_with_fee=fiat_in_with_fee,
+                            fiat_fee=fiat_fee,
+                            notes=key,
                         )
                     )
                 else:
-                    kwargs.update({
-                        'crypto_out_no_fee': str(amount),
-                        'crypto_out_with_fee': str(amount + RP2Decimal(record[_FEE])),
-                        'fiat_out_no_fee': str(RP2Decimal(trade_history[record[_REFID]][_COST]) - RP2Decimal(
-                            trade_history[record[_REFID]][_FEE])),
-                        'is_spot_price_from_web': False,
-                    })
+                    crypto_out_no_fee: str = str(amount)
+                    crypto_out_with_fee: str = str(amount + RP2Decimal(record[_FEE]))
+                    fiat_out_no_fee: str = str(RP2Decimal(trade_history[record[_REFID]][_COST]) - RP2Decimal(
+                        trade_history[record[_REFID]][_FEE]))
+                    is_spot_price_from_web: bool = False
```

Comment:
`is_spot_price_from_web` isn't actually necessary. The `pair_converter` will set it for you.